### PR TITLE
Restore BE district scraper

### LINKED ISSUE #1816: New format Bern District numbers
Unfortunately only Incidence numbers and only from the last week, older ones are not in the csv:
https://raw.githubusercontent.com/openDataBE/covid19Data/develop/7_d_inzidenz_verwaltungskreis.csv

Any way to get this in a way so it is consistent with the existing data?

### PR DESCRIPTION
**Pull request by maekke**
@metaodi the computation from the "7 Tage Inzidenz" back to the absolute number of cases seem to produce pretty reasonable results (for example Bern-Mittelland ` 23.43 / 100000 * = 96.9809874`)

Resolves #1816

### CODE REVIEW

File: scrapers/scrape_be_districts.py [hunk 1]
```diff
@@ -1,79 +1,39 @@
...
+    district_id = int(row['bfs_nummer'])
+    dd = sc.DistrictData(district=district_ids[district_id], canton='BE')
+    dd.url = url
+    dd.district_id = district_id
+    dd.population = row['einwohnerzahl']
+    date = sc.date_from_text(row['datum'])
+    week = date.isocalendar()[1]
+    dd.week = week
+    dd.year = date.year
+    dd.new_cases = round(float(row['7_d_inzidenz']) / 100e3 * float(row['einwohnerzahl']))
```
Comment (metaodi) [inline scrapers/scrape_be_districts.py:38]:
```suggestion
    dd.new_cases = round(float(row['7_d_inzidenz']) / 100e3 * int(row['einwohnerzahl']))
```

The result of this should be an integer, right? Or is there something I'm missing?

Comment (maekke) [reply scrapers/scrape_be_districts.py]: Yep, it's an integer - you are of course right! I've applied your suggestion, thank you.

**Comment by balzamas** [comment]
It seems that the data is updated daily now, not weekly. 
https://github.com/openZH/covid_19/blob/master/fallzahlen_bezirke/fallzahlen_kanton_BE_bezirk.csv has now week 23 even though the week just begun. I checked the history of the source csv from  @openDataBE and it indeed is a daily update.

**Comment by metaodi** [comment]
@balzamas I created a new PR #1819 to address this.

**Pull request merged**